lxd/device/usb.go

```diff
@@ -173,7 +173,7 @@ func (d *usb) startVM() (*deviceConfig.RunConfig, error) {
 	for _, usb := range usbs {
 		if usbIsOurDevice(d.config, &usb) {
 			runConf.USBDevice = append(runConf.USBDevice, deviceConfig.USBDeviceItem{
-				DeviceName:     fmt.Sprintf("%s-%d", d.name, len(runConf.USBDevice)),
+				DeviceName:     fmt.Sprintf("%s-%03d-%03d", d.name, usb.BusNum, usb.DevNum),
```

Comment:
A comment here summarising your excellent commit message as a warning/flag to not arbitrarily change this in the future and explain why its done like this would be excellent.